saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -103,6 +104,10 @@ def clean_input(cls, info, instance, data):
         redirect_url = data.pop("redirect_url", None)
         channel_id = data.pop("channel_id", None)
 
+        shipping_method = get_shipping_model_by_object_id(
```

Comment:
IMHO, the name can be incorrect 🤔 .  This function returns an instance of ShippingMethod (if exists). Is it something dedicated for EP? I don't see any handler for shipping method from EP. Also if this case is specific only for DraftorderUpdate it should be placed there, not in DraftOrderCreate

Reply:
This logic is for both DraftorderUpdate and DraftOrderCreate mutations and because we don't have a connection between the ShippingMethod model and the type class we cannot use the build-in `get_node_or_error` function.

Reply:
Instead of adding a special function for handling this case, what do you think about using `get_node_or_error` but with a QS from ShippingMethod?
`get_node_or_error(id, ..., qs=ShippingMethod.objects)` 
@maarcingebala  what do you think?

Reply:
If possible I would really try to avoid adding new functions just to get items from DB by ID, we already have at least three functions for that.